src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/blocks/leftcurly/InputLeftCurlyTestSwitchExpressionsNewLine.java

```diff
@@ -11,7 +11,7 @@
  * LITERAL_FOR , LITERAL_IF , LITERAL_SWITCH , LITERAL_SYNCHRONIZED ,
```

Comment:
this tokens can be removed, since they were not set explicitly inside test case

Reply:
Done👍